### ISSUE TITLE
Station indexing.

### ISSUE THREAD

**Issue by mjpdenver** [CoWy-ASA/RiverWatch#3]
How do we identify the relative location of sampling sites within the river basin. We have asked Barb a question about this and Karl from the USGS has provided some general information. Alternatively, it was suggested that site numbering could be done using ArcGIS tools.

**Comment by hornb** [comment]
I am not sure if I understand the question or if it is not too late.  In our station table we have the following fields that may or may not help. Each station has a unique id, called station number, that no other station can have in the database.  85% of all stations are on Rivers, some are on lakes, wells, adits and other discharges but they can be filtered out using name primarily.

Station Name: \* Name is required. 
 Station Number: \* Station Number is required. 
 Water Code: \*  segmentation used by CPW to identified management of fisheries 
 River:   Name of River
 Description:  driving directions
 Station Type: \*  chemical, phys or biological
 Station Status: \*  
 Aquatic Model Index:  an id to sync with CPW fisheries database
 Water Body ID: \*   COARCI01, an example - water segments that discharge permits are regulated on
 Range:  
 Section:  
 Quarter Section:  
 Grid:   from days pre GIS
 Station QUAD:   7.5 minute quad name
 RiverWatch Watershed:   8 primary
 WQCC Watershed:   about 30 choices
 Hydro Unit:  8-digit
 Eco Region:  
 Elevation:  
 Watershed Report:  
 Longitude: \* Longitude is required. 
 Latitude: \* Latitude is required. 
 UTMX:  
 UTMY:  
 County:  
 State:  
 Near City:  
 Move:  text field
 Region:   12N 13N 14N 
 USGS:   (flow gauge)
 State Engineering:  (flow gauge)
 Comment:

**Comment by mjpdenver** [comment]
I don't think this information quite gets at the issue.  Also, this task is probably easier  than I am making it out to be.  Manually, a glance at at map and you can tell how sites are ordered.

Here is a note I sent to Karl at the USGS.

(https://github.com/CoWy-ASA/RiverWatch/wiki/USGS-Water-Flow-Information)

Basically -  if I have a list of sites and I picked a site, I would like  to be able to identify the up- and down- stream sites.  Obviously this would be trivial on a single stretch of river - but gets more complicated with river branches.
